libs/core/algorithms/tests/regressions/for_each_datapar.cpp

```diff
@@ -4,6 +4,7 @@
 //  Distributed under the Boost Software License, Version 1.0. (See accompanying
 //  file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
 
+#include <hpx/include/datapar.hpp>
```

Comment:
```suggestion
#include <hpx/local/datapar.hpp>
```
The full/include module is not available if the distributed runtime is disabled.